### ISSUE TITLE
Missing cost estimate for `azurerm_postgresql_flexible_server` compute GP_Standard_D2ds_v5

### ISSUE THREAD

**Issue by ljetten** [infracost/infracost#3012]
Hello,

The cost estimate for GP_Standard_D2ds_v5 postgres flexible server comes up with a warning: 
```
WRN No products found for module.<name>.azurerm_postgresql_flexible_server.main Compute (GP_Standard_D2ds_v5), using 0.00
```
I have tried using infracost v0.10.10, v0.10.33 and v0.10.35 and all have the same result. 
Other tiers do seem to work fine.

Let me know if you need more info, thanks!

**Comment by vdmgolub** [comment]
FYI, we'll keep this issue open until the next release :)

**Comment by hugorut** [comment]
Thanks @ljetten, a fix for this is now live in the [v0.10.36](https://github.com/infracost/infracost/releases/tag/v0.10.36) release.
